app/assets/javascripts/services/queriesSvc.js

```diff
@@ -77,6 +107,28 @@ angular.module('QuepidApp')
         }
       }
 
+      function normalizeDocExplains(query, searcher, fieldSpec) {
+        var normed = [];
+
+        if (searcher.type === 'es') {
+          normed = esExplainExtractorSvc.docsWithExplainOther(searcher.docs, fieldSpec);
+        } else {
```

Comment:
likewise on the es versus solr..   this isn't really critical cause we'll ahve to do an overhaul anyway when we add OpenSearch!